.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -316,6 +325,39 @@ task ImportGISAID {
     export PYTHONUNBUFFERED=true
     python3 /usr/src/app/aspen/workflows/import_gisaid/save.py       \
             --metadata-file ~{gisaid_metadata} 1>&2
+    echo done > gisaid_import_complete
+    >>>
+
+    output {
+        String gisaid_import_complete = read_string("gisaid_import_complete")
+    }
+
+    runtime {
+        docker: docker_image_id
+    }
+}
+
+task ImportLocations {
+    input {
+        String docker_image_id
+        String aws_region
+        String aspen_config_secret_name
+        String remote_dev_prefix
+        String gisaid_import_complete
+        File gisaid_metadata
+    }
+
+    command <<<
+    set -Eeuo pipefail
+    aws configure set region ~{aws_region}
+
+    export ASPEN_CONFIG_SECRET_NAME=~{aspen_config_secret_name}
+    if [ "~{remote_dev_prefix}" != "" ]; then
+        export REMOTE_DEV_PREFIX="~{remote_dev_prefix}"
+    fi
+
+    export PYTHONUNBUFFERED=true
```

Comment:
small nit but i feel like setting this to `false` makes more sense since we want stdout and stderr streams to be unbuffered

Reply:
If we want the streams to be unbuffered, then we should set this to `true`. (The var is PYTHON + **UN**BUFFERED).

Reply:
oh whoops i read that as `PYTHONBUFFERED`, my mistake